packages/wxt/src/core/utils/building/generate-wxt-dir.ts

```diff
@@ -73,7 +77,26 @@ async function writeImportsEslintFile(
     .forEach((name) => {
       eslintrc.globals[name] = options.eslintrc.globalsPropValue;
     });
-  await fs.writeJson(options.eslintrc.filePath, eslintrc, { spaces: 2 });
+
+  const jsonFilePath = options.eslintrc.filePath;
+  await fs.writeJson(jsonFilePath, eslintrc, { spaces: 2 });
+
+  await writeEslintNineFile(majorEslintVersion, jsonFilePath);
+}
+
+async function writeEslintNineFile(
+  majorEslintVersion: number,
+  jsonFilePath: string,
+) {
+  if (majorEslintVersion >= 9) {
```

Comment:
This if statement should go outside this function. We should only call `writeEslintNineFile` if it is eslint 9. Small readibility thing.

Reply:
Fair enough, I looked at it and went "you know an early return would make this easier to read, or could have it outside the function"